src/components/custom/CustomSelectInput.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect } from 'react'
+import 'react-responsive-select/dist/react-responsive-select.css'
+import i18n from '../../i18n'
+import {
+  Select,
+  CaretIcon,
+  MultiSelectOptionMarkup,
+  ModalCloseButton,
+} from 'react-responsive-select'
+
+const CustomSelectInput = ({ locationValue, setInputValue, disabled }) => {
+  const handleChange = (newValue) => {
+    if (newValue.options[0].value) {
+      setInputValue(newValue.options.map((location) => location.value))
+    } else {
+      setInputValue(locations.slice(1).map((location) => location.value))
+    }
+  }
+
+  useEffect(() => {
+    if (i18n.language === 'Svenska') {
+      locations[0] = {
+        value: 'null',
+        text: 'Alla',
+        markup: <MultiSelectOptionMarkup text='Alla' />,
+      }
+    } else {
+      locations[0] = {
+        value: 'null',
+        text: 'Any',
+        markup: <MultiSelectOptionMarkup text='Any' />,
+      }
+    }
+    // eslint-disable-next-line
```

Comment:
is this needed now?

Reply:
Yes or we get a compiler warning which will mess up netlify